### PR TITLE
swap to pioneer flag once claimed

### PR DESCRIPTION
**Pull request by Dasku**

### CODE REVIEW

File: creep.action.attackController.js [hunk 1]
```diff
@@ -23,8 +23,20 @@ action.newTarget = function(creep){
     if( !creep.flag.room || creep.flag.pos.roomName != creep.pos.roomName){
         return creep.flag;
     }
-
-    return creep.flag.room.controller;
+    if( creep.flag.room.controller.my ) { 
+        // already claimed, change flag
```
Comment (cyberblast) [inline creep.action.attackController.js]: maybe we could add a second secondary color to specify if we want to claim or reserve the liberated room...

Comment (Dasku) [inline creep.action.attackController.js]: That's a good idea. I'll update this when I get home from work

Comment (Dasku) [inline creep.action.attackController.js]: If we reserve it, do we change the flag to mine? or reserve? (reserve feels like a waste)

Comment (cyberblast) [inline creep.action.attackController.js]: could also be a tactical reserve

Comment (Dasku) [inline creep.action.attackController.js]: Can do both sure

**Pull request closed without merging**